### DISCUSSION TITLE
The user which is used to deploy to GitHub Pages might be wrong.

### DISCUSSION THREAD

**Discussion by yude** [upptime/upptime#164] [Q&A] [answered]
It seems that my account "yude" is used to deploy the site. Like this:
![chrome_20210103_123821_738](https://user-images.githubusercontent.com/14804458/103471191-9394e380-4dc0-11eb-8714-ed6ab62a7fae.png)
I think upptime-bot should be used to do this. (and we can avoid being counted these commits on GitHub.)
I'm not native of English and I'm sorry if there're any mistakes. Thank you.

**Comment by AnandChowdhary** [comment, marked as answer]
We're discussing this here: https://github.com/upptime/upptime/issues/118. It seems like a GitHub issue currently out of our hands, but I'm happy to hear any thoughts over at https://github.com/upptime/upptime/issues/118.

**Reply by yude** [reply]
Thank you for reply!
